test/car-route-test.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+const Car = require('../model/car.js');
+const PORT = process.env.PORT || 8000;
+
+process.env.MONGODB_URI = 'mongodb://localhost/carappdev';
+
+require('../server.js');
+
+const url = `http://localhost:${PORT}`;
+const exampleCar = {
+  make: 'test car make',
+  model: 'test car model'
+};
+
+describe('Car Routes', function() {
+  describe('POST: /api/car', function(){
+    describe('with a valid body', function() {
+      after( done => {
+        if (this.tempCar) {
+          Car.remove({})
+          .then( () => done())
+          .catch(done);
+          return;
+        }
+        done();
+      });
+
+      it('should return a car', done => {
+        request.post(`${url}/api/car`)
+        .send(exampleCar)
+        .end((err, res) => {
```

Comment:
Make sure you are handling an err if it comes back.  'if (err) return done(err);'